src/views/virtualmachines/details/tabs/configuration/storage/components/tables/disk/DiskRow.tsx

```diff
@@ -73,7 +74,13 @@ const DiskRow: React.FC<
         className="dropdown-kebab-pf pf-v5-c-table__action"
         id=""
       >
-        <DiskRowActions diskName={obj?.name} pvcResourceExists={isPVCSource} vm={vm} vmi={vmi} />
+        <DiskRowActions
+          diskName={obj?.name}
+          isPVCBound={isPVCStatusBound(obj?.sourceStatus)}
+          pvcResourceExists={isPVCSource}
```

Comment:
I think moving obj as a single prop and extracting diskName, isPVCBound and pvcResourceExists inside DiskRowActions is better then sending 3 props. what do u think?

Reply:
That's a good improvement I think. Thanks!